pubsub/aws/snssqs/metadata.yaml

```diff
@@ -0,0 +1,160 @@
+    type: number
+    default: '1'
+    example: '1'
+  - name: messageMaxNumber
+    required: false
+    description: |
+      Maximum number of messages to receive from the queue at a time.
+    type: number
+    default: '10'
+    example: '10'
+  - name: fifo
+    description: |
+      Use SQS FIFO queue to provide message ordering and deduplication.
+      See SQS FIFO documentation further details.
+    url:
+      title: "SQS FIFO documentation"
+      url: "https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/FIFO-queues.html"
+    type: bool
+    default: 'false'
+    example: '"true", "false"'
+  - name: fifoMessageGroupID
+    required: false
+    description: |
+      If fifo is enabled, instructs Dapr to use a custom Message Group ID 
+      for the pubsub deployment. This is not mandatory as Dapr creates a
+      custom Message Group ID for each producer, thus ensuring ordering
+      of messages per a Dapr producer.
+      See Message Group ID Property documentation.
+    url:
+      title: "Message Group ID Property documentation"
+      url: "https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/using-messagegroupid-property.html"
+    example: '"app1-mgi"'
+    type: string
+  - name: disableEntityManagement
+    description: |
+      When set to true, SNS topics, SQS queues and the SQS subscriptions to
+      SNS do not get created automatically.
+    type: bool
+    default: 'false'
+    example: '"true", "false"'
+  - name: disableDeleteOnRetryLimit
+    description: |
+      When set to true, after retrying and failing of messageRetryLimit
+      times processing a message, reset the message visibility timeout
+      so that other consumers can try processing, instead of deleting
+      the message from SQS (the default behvior).
+    type: bool
+    default: 'false'
+    example: '"true", "false"'
+  - name: assetsManagementTimeoutSeconds
+    required: false
+    description: |
+      Amount of time in seconds, for an AWS asset management operation,
+      before it times out and cancelled. Asset management operations 
+      are any operations performed on STS, SNS and SQS, except message
+      publish and consume operations that implement the default Dapr
+      component retry behavior. The value can be set to any non-negative
+      float/integer.
+    type: number
+    default: '1'
+    example: '0.5, 10'
+  - name: concurrencyMode
+    required: false
+    description: |
+      When messages are received in bulk from SQS, call the subscriber
+      sequentially (“single” message at a time), or
+      concurrently (in “parallel”).
+    default: '"parallel"'
+    example: '"single", "parallel"'
+    type: string
+  - name: accountId
+    required: false
+    description: |
+      The AWS account ID. Resolved automatically if not provided.
+    example: '""'
+    type: string
+  - name: consumerId
```

Comment:
I think this can be omitted as it's a built-in property for all pubsub https://github.com/dapr/components-contrib/blob/master/.build-tools/pkg/metadataschema/validators.go#L145-L156